.github/actions/setup-python/action.yml

```diff
@@ -40,6 +40,13 @@ runs:
         path: .venv
         key: "venv-${{ runner.os }}-${{ runner.environment }}-${{ hashFiles('requirements**.lock') }}"
 
+    - name: Export PYO3 Path
+      shell: bash
+      # Figure out a way to dynamically get python version
+      run: |
+        echo "PYO3_PYTHON=/home/runner/work/vortex/vortex/.venv/bin/python" >> $GITHUB_ENV
+        echo "PYO3_ENVIRONMENT_SIGNATURE=cpython-3.11-64bit" >> $GITHUB_ENV
```

Comment:
can presumably remove this from https://github.com/spiraldb/vortex/blob/develop/.github/actions/setup-python/action.yml#L51

Reply:
yes, it's removed